Actions/DownloadProjectDependencies/DownloadProjectDependencies.Action.ps1

```diff
@@ -40,12 +40,16 @@ function DownloadDependenciesFromCurrentBuild($baseFolder, $project, $projectsDe
             $dependencyBuildMode = 'Default';
         }
 
-        $currentBranch = $ENV:GITHUB_REF_NAME
+        $headBranch = $ENV:GITHUB_HEAD_REF
+        # $ENV:GITHUB_HEAD_REF is specified only for pull requests, so if it is not specified, use GITHUB_REF_NAME
+        if (!$headBranch) {
+            $headBranch = $ENV:GITHUB_REF_NAME
+        }
 
-        $baseBranch = $ENV:GITHUB_BASE_REF_NAME
-        # $ENV:GITHUB_BASE_REF_NAME is specified only for pull requests, so if it is not specified, use the current branch
+        $baseBranch = $ENV:GITHUB_BASE_REF
+        # $ENV:GITHUB_BASE_REF is specified only for pull requests, so if it is not specified, use GITHUB_REF_NAME
         if (!$baseBranch) {
-            $baseBranch = $currentBranch
+            $baseBranch = $GITHUB_REF_NAME
```

Comment:
Shouldn't this be $ENV:GITHUB_REF_NAME ?   (also in the comment)

Reply:
Oh, yes, true. My bad.

Reply:
```suggestion
            $baseBranch = $ENV:GITHUB_REF_NAME
```